### ISSUE TITLE
Batchnorm in NN Head

### ISSUE THREAD

**Issue by ivan-chai** [dllllb/pytorch-lifestream#157]
There are two problems with nn.Head:
1. Linear layer when followed by BatchNorm must not include a trainable bias vector (bias=False).
2. Batchnorm needs a tensor with shape (B, C, L), while PTLS uses (B, L, C). The conversion between these two formats must be added to the forward method.

**Comment by ivkireev86** [comment]
@ivan-chai, hello

As far as I understand, BatchNorm in the Head class works not with the size `(B, L, C)` but with the size `(B, H)`? What use case do you have in mind?
